README.md

```diff
@@ -9,8 +9,13 @@ Gopher images from [Ashely McNamara's fantastic collection](https://github.com/a
 
 ## Installation
 
-Maybe try installing the `space-gophers.saver` file directly? I don't really know enough about the security restrictions of macOS to say if it will work correctly. If not, you can build the screensaver as outlined in the Development Notes section. Pull requests and suggestions on how to make this easier to install are welcome.
+Due to macOS security restrictions you cannot directly install this if you have Gatekeeper active (it requires this to be signed by a valid developer certificate). However, you can remove the `com.apple.quarantine` attribute to allow installation, like this:
 
+```
+xattr -d "com.apple.quarantine"  space-gophers.saver
```

Comment:
Can we just do this to the `space-gophers.saver` that we include in the repo? That way people don't need to worry about it.

Also, if that makes it possible to create a `.saver` that can be double-clicked to install on any machine, maybe we can work this into github's releases list so people don't need to clone the repo just to get the screensaver itself.

Reply:
I'm not sure might be possible to have it fly under the radar if it's done to the file in the repo, and the end-user downloads the repo using `git clone`. However if the `.saver` file or an archive is downloaded directly macOS will apply this attribute to it and all files extracted by the archive.

The "correct" way to solve this is by being a registered developer and sign the `.saver` file.. but that costs $99/y. 😏

Reply:
aah, fair enough.